dicomImport/matRad_importDicomCt.m

```diff
@@ -197,16 +198,24 @@
 ct.dicomInfo.Height                  = ctInfo(1).Height;
 ct.dicomInfo.RescaleSlope            = ctInfo(1).RescaleSlope;
 ct.dicomInfo.RescaleIntercept        = ctInfo(1).RescaleIntercept;
-if isfield(completeDicom,'PatientName')
+if isfield(completeDicom,'PatientName') && dicomBool == true
     ct.dicomInfo.PatientName         = completeDicom.PatientName;
 end
 
+% do not remove the complete dicom information here because ...
+% calcWaterEqD needs some information
 ct.dicomInfo_org = completeDicom;
+
```

Comment:
No, actually this field is removed later on (line 216-218) so in case the checkbox is unchecked, at the end there is no backdoor to get the patient information. But I agree it's a rather dirty solution. 
I will just keep the information relevant for matRad in dicomInfo (not in dicomInfo_org) and I think I may rename the complete dicom field here and in pln as well. Maybe dicomInfoBackup or dicomInfoMeta. Any suggestions?